main.py

```diff
@@ -33,6 +37,32 @@ def cli_entrypoint(args):
     exit(1)
 
 
+def __build_configuration(config_file: str):
+  file_config = {}
+  if os.path.exists(config_file):
+    with open(config_file, "r", encoding="utf-8") as f:
+      file_config = json.loads(f.read())
+    if file_config.get("red_key") == "xxxxxxxxxx":
```

Comment:
since you want the config file to overwrite env vars, we need to check for default config values